lib/service/servicecfg/windows.go

```diff
@@ -162,4 +172,9 @@ type LDAPConfig struct {
 	ServerName string
 	// CA is an optional CA cert to be used for verification if InsecureSkipVerify is set to false.
 	CA *x509.Certificate
+	// LocateServer automatically locates the LDAP server using DNS SRV records.
+	// https://ldap.com/dns-srv-records-for-ldap/
+	LocateServer
+	// Logger is the logger for the service.
+	Logger *slog.Logger
```

Comment:
Is this used anywhere? I'm not seeing it.

Reply:
Ah yeah, that's only required in `ldap.go`